pyproject.toml

```diff
@@ -224,6 +225,7 @@ minversion = "6"
 # error if not installed
 required_plugins = [
```

Comment:
Now that we use `pytest` virtually everywhere, could you remove the comments on lines 221 and 224–225?